docs/defects-api.yml

```diff
@@ -39,6 +38,69 @@ paths:
           description: Access token does not have the required scope
         "404":
           description: Defects not found
+
+  /defects/iva/getDefectsForVehicle:
```

Comment:
Are we okay with this naming with get in the URL and not the standard?
 ```suggestion
   /defects/iva:
```

Reply:
Would /defects/iva/vehicle be more preferred?

Reply:
Like this change a lot :)

Reply:
As in what you have done, not the /vehicle, sorry!